lib/browser/api/app.js

```diff
@@ -77,6 +77,24 @@ app.allowNTLMCredentialsForAllDomains = function (allow) {
   }
 }
 
+const makeSingleInstance = app.makeSingleInstance
+app.makeSingleInstance = function (callback) {
+  return makeSingleInstance.call(app, function (argv, ...args) {
+    if (process.platform === 'win32') {
+      // Delay the checking so the callback is run as quickly as we would expect
+      setImmediate(() => {
+        argv.forEach((arg) => {
+          const protocolMatch = arg.match(/^([\s\S]+):\/\/[\s\S]+/)
```

Comment:
oh sorry, so early feedback, just another 5 cents, maybe there is an `Event` mock somewhere on npm as well, so if someone goes to check event.type or something it will be confusing again if its not there.